axiom/monitors.go

```diff
@@ -106,40 +106,47 @@ func (c *MonitorType) UnmarshalJSON(b []byte) (err error) {
 }
 
 type Monitor struct {
+	// Identification
 	// ID is the unique ID of the monitor.
 	ID string `json:"id,omitempty"`
+	// CreatedAt is the time when the monitor was created.
+	CreatedAt time.Time `json:"createdAt"`
+	// CreatedBy is the ID of the user who created the monitor.
+	CreatedBy string `json:"createdBy"`
+	// Name is the name of the monitor.
+	Name string `json:"name"`
 	// Type sets the type of monitor. Defaults to [Threshold]
 	Type MonitorType `json:"type"`
+	// Description of the monitor.
+	Description string `json:"description,omitempty"`
+
+	// Configuration
```

Comment:
```suggestion
	// Configuration

```

Otherwise it looks odd in GoDoc.